modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -29,6 +29,12 @@ $ firewall-cmd --permanent --zone dockerc --add-port 443/tcp
 $ firewall-cmd --reload
 ----
 
+WARNING: Due to a number of services using ports which on Linux are restricted to root users only. This may result in issues if you try to interact with services on a local cluster from an external device. If you encounter these issues, you can add the ports to your firewall. Depending on the port your service uses, an example of the ports you may want to add to your firewall are:
```

Comment:
Some {product-name} use ports that are restricted on Linux. If you encounter issues connecting to {product-name} from external devices, add the required ports to your firewall. For example: